helm/csi-isilon/values.yaml

```diff
@@ -329,6 +329,15 @@ isiPath: /ifs/data/csi
 # Examples: "0777", "777", "0755"
 isiVolumePathPermissions: "0777"
 
+# ignoreHosts: Ignore unresolvable hosts on the OneFS
+# When set to true, OneFS allows new host to add to existing export list though any of the existing hosts from the
+# same exports are unresolvable/doesn't exist anymore.
+# Allowed values:
+#   true: ignore existing unresolvable hosts and append new host to the existing export
+#   false: exhibits OneFS default behavior i.e. if any of existing hosts are unresolvable while adding new one it fails
+# Default value: false
+ignoreHosts: false
```

Comment:
rename this param to ignoreUnresolvableHosts, to be inline with OneFS API param name

Reply:
Also rename the new env var accordingly